### ISSUE TITLE
changing the interactions crashes the game

### ISSUE THREAD

**Issue by Kagura101** [Brionjv/Tomodachi-Life-Save-Editor#141]
so all i did was change the fight interaction to a huge fight interaction but when i restore the save file, it crashed the game.
Before:
![image](https://user-images.githubusercontent.com/95944842/181377367-8a6fe152-e6d6-4918-b366-3870472c8426.png)

After: 
![image](https://user-images.githubusercontent.com/95944842/181377650-91cff6f2-728d-480c-ac34-eaae1666e2f8.png)

**Comment by Brionjv** [comment]
Bartz have same interaction ?

**Comment by Kagura101** [comment]
yes, i made sure they both have the same interaction, in fact i'll send my save data to check between those 2 saves, note all i did was to change the fight interaction

BEFORE (NORMAL FIGHT)
[savedataArc.txt](https://github.com/Brionjv/Tomodachi-Life-Save-Editor/files/9212956/savedataArc.txt)

AFTER (HUGE FIGHT)
[savedataArc.txt](https://github.com/Brionjv/Tomodachi-Life-Save-Editor/files/9212958/savedataArc.txt)

**Comment by Brionjv** [comment]
you have lot of babies interactions, these interactions are unstable for now, you need to delete them to fix crashes
![Sans titre 2](https://user-images.githubusercontent.com/22226393/181712029-c472a012-4707-4a70-a3f1-c91e79b0643a.png)

**Comment by Kagura101** [comment]
Okay so it's been a few days since the last post, so the fight previosly ended and I resolved/deleted a lot of babies interactions and the game still crashes and yes, i made sure both Jared and Honey have the same interaction and all i did was to change to normal fight to a HUGE fight.

BEFORE: [savedataArc.txt](https://github.com/Brionjv/Tomodachi-Life-Save-Editor/files/9245660/savedataArc.txt)

AFTER:
[savedataArc.txt](https://github.com/Brionjv/Tomodachi-Life-Save-Editor/files/9245664/savedataArc.txt)

**Comment by Brionjv** [comment]
I don't know why game crash. I'll try to found

**Comment by Kagura101** [comment]
hey uhh any updates regarding my situation?

**Comment by Brionjv** [comment]
for now no, changing fight to huge fight corrupted save file. I don't know why
